github_connector/models/github_analysis_rule_info.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2020: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class GithubAnalysisRuleInfoMixin(models.AbstractModel):
+    _inherit = "github.analysis.rule.mixin"
+    _name = "github.analysis.rule.info.mixin"
+    _description = "Github Analysis Rule Mixin"
+
+    code_count = fields.Integer(string="# Code")
+    documentation_count = fields.Integer(string="# Documentation")
```

Comment:
For keeping a similar column width
```suggestion
    documentation_count = fields.Integer(string="# Doc.")
```

Reply:
Changes applied now